common/src/Microsoft.Azure.IIoT.Messaging.SignalR/src/Services/SignalRServiceHost.cs

```diff
@@ -185,6 +194,23 @@ public class SignalRServiceHost : IIdentityTokenGenerator, IEndpoint,
             return _hub.UserGroups.RemoveFromGroupAsync(client, group, ct);
         }
 
+        /// <inheritdoc/>
```

Comment:
Best to remove the comment or add a full comment for the private method

Reply:
done